bliss/encoder.py

```diff
@@ -385,7 +409,8 @@ def _generic_step(self, batch, logging_name, log_metrics=False, plot_images=Fals
 
     def training_step(self, batch, batch_idx, optimizer_idx=0):
         """Training step (pytorch lightning)."""
-        return self._generic_step(batch, "train")
+        do_data_augmentation = self.do_data_augmentation
```

Comment:
Can you do the data augmentation here in `training_step` rather than in `_generic_step`? (We don't want to do data augmentation unless we're training.)